packages/gunshi/src/types.ts

```
@@ -234,23 +241,27 @@ export interface CliOptions<G extends GunshiParamsConstraint = DefaultGunshiPara
    * Hook that runs before any command execution
    * @param ctx - The command context
+   * @since v0.24.0
    */
   onBeforeCommand?: (ctx: Readonly<CommandContext<G>>) => Awaitable<void>
   /**
    * Hook that runs after successful command execution
    * @param ctx - The command context
    * @param result - The command execution result
+   * @since v0.24.0
    */
   onAfterCommand?: (ctx: Readonly<CommandContext<G>>, result: string | void) => Awaitable<void>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)